uint64 vehicleID
GPSSample gpsSample
